### ISSUE TITLE
Ideas for knowledge resources to add

### ISSUE THREAD

**Issue by StefKirsch** [UB-Mannheim/FAIR-GPT#3]
## Softwrae
- [The Turing Way](https://the-turing-way.netlify.app/index.html)
- [Choose a License (for software)](https://choosealicense.com/)
- [Code Refinery](https://coderefinery.org/)
- [Fair Software.eu](https://fair-software.eu/)
- [Practical guide to Software Management Plans](https://zenodo.org/records/7589725)

## Others
- [Horizon Europe](https://research-and-innovation.ec.europa.eu/funding/funding-opportunities/funding-programmes-and-open-calls/horizon-europe_en)
- [The Carpentries](https://carpentries.org/index.html)
- [Practical Guide to the International Alignment of Research Data Management](https://www.scienceeurope.org/media/jezkhnoo/se_rdm_practical_guide_final.pdf)

**Comment by renat-kaufmann** [comment]
Things to clarify:
* Licenses of resources: How shall FAIR GPT attribute the sources of knowledge in case of CC BY licenses?
* API vs Uploaded Knowledge: I would prefer to add external API (if available) because the uploaded files will slow down FAIR GPT. Or maybe the instructions should say in which case which file to use, to optimize the performance. 
* Size and type of uploaded files: Not sure yet what is the maximal size and maximal number of uploaded files. What would be preffered format for performance. I see [this discussion](https://community.openai.com/t/gpts-best-file-format-for-knowledge-to-feed-gpts/497368).

**Comment by StefKirsch** [comment]
> Things to clarify:
> 
> * Licenses of resources: How shall FAIR GPT attribute the sources of knowledge in case of CC BY licenses?
> * API vs Uploaded Knowledge: I would prefer to add external API (if available) because the uploaded files will slow down FAIR GPT. Or maybe the instructions should say in which case which file to use, to optimize the performance.
> * Size and type of uploaded files: Not sure yet what is the maximal size and maximal number of uploaded files. What would be preffered format for performance. I see [this discussion](https://community.openai.com/t/gpts-best-file-format-for-knowledge-to-feed-gpts/497368).

Yeah, I'm worried about it too that it all might get a bit cluttered and the GPT would have a hard time finding the right resources. I am not that familiar with RAG and how it's implemented for chatGPT. Maybe it's a good idea to have a call and go over some technical aspects?

**Comment by renat-kaufmann** [comment]
Let's move to emails (see [here](https://www.bib.uni-mannheim.de/ihre-ub/ansprechpersonen/dr-renat-shigapov)) for that. Please propose a couple of date/times.
